tests/data/data_manager/test_foldermap.py

```diff
@@ -125,6 +127,38 @@ class TestFolderMapView:
                     ),
                 ],
             ),
+            (
+                {"missing_default": DEFAULT, "molname": "O2", "basis": FULL, "bondlength": ["0.6"]},
+                [
+                    (
+                        {"molname": "O2", "basis": "STO-3G", "bondlength": "0.6"},
+                        "qchem/O2/STO-3G/0.6.h5",
+                    ),
+                ],
+            ),
+            (
+                {
+                    "missing_default": DEFAULT,
+                    "molname": "O2",
+                    "basis": FULL,
+                    "bondlength": ["0.6", "200"],
+                },
```

Comment:
is this the test-case that fails without the change/was it failing locally without the fix?

Reply:
The one just above it actually - the [ticket](https://app.shortcut.com/xanaduai/story/40483/bug-qml-data-load-fails-when-using-full-parameter-value) explains it.